mock-api/msw/db.ts

```diff
@@ -55,10 +55,24 @@ function ensureNoParentSelectors(
   }
 }
 
+export const resolveIpPool = (poolName: string | undefined | null, context = '') => {
+  if (poolName) {
+    return lookup.ipPool({ pool: poolName })
+  }
+  try {
+    return lookup.siloDefaultIpPool({ silo: defaultSilo.id })
+  } catch (_error) {
+    const contextMsg = context ? ` ${context}` : ''
+    throw new Error(
+      `No IP pool specified${contextMsg} and no default IP pool configured for silo. Please specify a pool.`
+    )
+  }
+}
+
 export const getIpFromPool = (poolName: string | undefined | null) => {
```

Comment:
I see this was already like this, but I'm realizing this is not always a name so it might be helpful to rename `poolName`. In the lookup functions we call it `id` even when it might be a name, which is also kind of shitty, so maybe just go maximalist: `poolNameOrId`.